css/css-transitions/CSSTransition-effect.tentative.html

```diff
@@ -87,6 +93,38 @@
   assert_equals(getComputedStyle(div).left, '0px');
 }, 'After setting a transition\'s effect to null, a new transition can be started');
 
+// This is a regression test for https://crbug.com/992668, where Chromium would
+// crash if the running transition's effect was set to null and the transition
+// was interrupted before it could finish due to the null effect.
+promise_test(async t => {
+  const div = addDiv(t);
+  div.style.left = '0px';
+
+  div.style.transition = 'left 100s';
+  getComputedStyle(div).left;
+  div.style.left = '100px';
+
+  assert_equals(div.getAnimations().length, 1);
+
+  const transition = div.getAnimations()[0];
+  await transition.ready;
+
+  // The transition needs to have a non-zero currentTime for the interruption
+  // reversal logic to apply.
+  await singleFrame();
+  assert_not_equals(transition.currentTime, 0);
+  assert_not_equals(getComputedStyle(div).left, '0px');
```

Comment:
@stephenmcgruer This test is failing on Firefox because it produces `'0px'` here. I don't think there's anything in the spec requiring that it have moved past 0px by this point (e.g. due to pixel rounding).

Based on the comment above this block I don't believe this assertion is needed (only the first one is). Would you mind dropping it?

If it is needed, would you mind reworking the test to wait until the value of `left` is actually non-zero?

Thanks.

Reply:
Argh, sorry, you're totally right. That's what I get for being lazy and not testing on Firefox, sorry.

I'll fix this tomorrow when I get to work. It's a bit weird because its really a regression test for Chromium, but we're meant to land all tests as WPT these days so I try to at least make them applicable to all browsers (e.g. test some real behavior). The test should wait until `left` is non-zero.